draft-ietf-tokbind-https-06.xml

```diff
@@ -937,6 +965,42 @@ contexts. Other approaches are possible, but are outside the scope of this speci
           mechanisms.
         </t>
       </section>
+        <section title="Correlation" anchor="Correlation">
+          <t>
+            An application's communicating parties, that receive 
+            Token Bindings for TLS connections other than their own, 
+            obtain information about the application's other TLS 
+            connections.
+            The Token Binding IDs in these other Token Bindings can 
+            serve as correlation handles for
+            the endpoints of the other connections.
+            If the application is already aware of these other 
+            connections, then no additional information is being exposed. 
```

Comment:
I'm still confused here. Is the application something like a native app, which is acting as an https client? Are these "other connections" ones that the application makes? If both of those are true, then it's trivial to see that the application is always aware of these other connections. However, I suppose that's not what this means, so I'm wondering what the definitions are of "the application" and "other connections".